ringpop_test.go

```diff
@@ -139,6 +139,22 @@ func (s *RingpopTestSuite) TestHandleEvents() {
 	s.Equal(11, listener.EventCount(), "expected 11 total events to be recorded")
 }
 
+func (s *RingpopTestSuite) TestRingpopReady() {
+	s.False(s.ringpop.Ready())
+	// Create single node cluster.
+	s.ringpop.Bootstrap(&BootstrapOptions{
+		swim.BootstrapOptions{
+			Hosts: []string{"127.0.0.1:3001"},
```

Comment:
Yes, because calling Bootstrap with a host list that contains only yourself creates a single node cluster.